packages/blade/src/components/Typography/Text/Text.tsx

```diff
@@ -13,7 +13,13 @@ type TextCommonProps = {
   truncateAfterLines?: number;
   children: React.ReactNode;
   weight?: keyof Theme['typography']['fonts']['weight'];
-  _color?: BaseTextProps['color'];
+  /**
+   * Sets the color of the Text component
+   *
+   * ## Only for internal use
+   *
+   */
```

Comment:
Maybe we can make it more obvious:

```suggestion
  /**
   * **Internal only**: Sets the color of the Text component
   *
   */
```

Reply:
Okay sure.